addons/web/static/src/views/list/list_view.js

```diff
@@ -25,8 +26,21 @@ export class ListArchParser extends XMLParser {
         };
         const activeFields = {};
         const columns = [];
+        let buttonId = 0;
         this.visitXML(arch, (node) => {
-            if (node.tagName === "field") {
+            if (node.tagName === "button") {
+                columns.push({
+                    type: "button",
+                    id: buttonId++,
+                    icon: node.getAttribute("icon") || false,
+                    title: node.getAttribute("string") || false,
+                    clickParams: {
+                        name: node.getAttribute("name"),
+                        type: node.getAttribute("type") || "object",
+                    },
+                    classes: ["btn-link"],
```

Comment:
seems unnecessary here. Can this be specified in the arch? If not, you can hardcode it in template